src/Payment.res

```diff
@@ -330,14 +330,21 @@ let make = (~paymentMode, ~integrateError, ~logger) => {
   React.useEffect(() => {
```

Comment:
@aritro2002 can you please try this 
```
  React.useEffect(() => {
    let cardError = switch (
      isCardSupported->Option.getOr(true),
      isCardValid->Option.getOr(true),
      cardNumber->String.length == 0,
    ) {
    | (_, _, true) => ""
    | (true, true, _) => ""
    | (true, _, _) => localeString.inValidCardErrorText
    | (_, _, _) => CardUtils.getCardBrandInvalidError(~cardNumber, ~localeString)
    }
    if isCardValid->Option.isSome {
      setCardError(_ => cardError)
    }
    None
  }, [isCardValid, isCardSupported])
```

In CardUtils.res please define the below function
```
let getCardBrandInvalidError = (
  ~cardNumber,
  ~localeString: OrcaPaymentPage.LocaleStringTypes.localeStrings,
) => {
  switch cardNumber->getCardBrand {
  | "" => localeString.enterValidCardNumberErrorText
  | cardBrandValue => localeString.cardBrandConfiguredErrorText(cardBrandValue)
  }
}
```

Reply:
Also, please test it for all the supported card brands for both valid and invalid cards.